src/main/resources/openapi/PrintAPIs.yaml

```diff
@@ -375,6 +454,51 @@ components:
         - en
       default: en
 
+    TemporaryCertificateSummariesResponse:
+      title: TemporaryCertificateSummariesResponse
+      type: object
+      description: A response containing a collection of Temporary Certificate summaries
+      properties:
+        temporaryCertificates:
+          type: array
+          description: List of Temporary Certificate summaries for the application.
+          items:
+            $ref: '#/components/schemas/TemporaryCertificateSummary'
+      required:
+        - temporaryCertificates
+    TemporaryCertificateSummary:
+      title: TemporaryCertificateSummary
+      type: object
+      description: An object containing Temporary Certificate summary
+      properties:
+        certificateNumber:
+          type: string
+          pattern: '^[0-9AC-HJ-NP-RT-Z]{20}$'
+          description: The unique identifier of the Certificate. A 20 character alphanumeric string.
+          example: DEV1W0XDH1368LWE1J40
+        status:
+          $ref: '#/components/schemas/TemporaryCertificateStatus'
+        userId:
+          type: string
+          format: email
+          example: fred.blogs@some-domain.co.uk
+          description: UserId of the user who generated the Temporary Certificate
+        dateTime:
```

Comment:
Is this the `issueDate` @NathanRussellValtech ? If the answer is yes, naming it as `issueDate` will make it more clear in my opinion. `dateTime` seems too generic and I wasn't able to figure it out without reading the description.